catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/impl/operations/QueryOperations.java

```diff
@@ -204,6 +206,22 @@ QueryResponse doQuery(QueryRequest queryRequest, FederationStrategy strategy)
 
         LOGGER.debug("Calling strategy.federate()");
 
+        Query originalQuery = queryRequest.getQuery();
+
+        long timeout = originalQuery.getTimeoutMillis();
+
+        Query modifiedQuery = new QueryImpl(originalQuery,
```

Comment:
Nope. This actually needs to be removed. It was leftover when I was adding in a default timeout. My bad